test/unit_tests/braket/pulse/test_waveforms.py

```diff
@@ -57,10 +57,10 @@ def test_arbitrary_wf_eq():
         assert wf != wfc
 
 
+@pytest.mark.xfail(raises=TypeError)
 def test_arbitrary_waveform_not_castable_into_list():
     amps = 1
-    with pytest.raises(TypeError):
-        wf = ArbitraryWaveform(amps)
```

Comment:
Why change this? `pytest.raises` is better than `xfail`, and we should actually be going the other way for all of our tests.